lib/tasks/export.rake

```diff
@@ -60,4 +61,20 @@ namespace :export do
     exporter = CicognaraMarc.new(cico_collection_id: coll, out_dir: output_dir)
     exporter.run
   end
+
+  desc "Export IIIF manifest links to PULFA DAOs"
```

Comment:
This one we intend to schedule via whenever/cron? Is there a follow-up ticket for that?

Reply:
Yes, I created https://github.com/pulibrary/figgy/issues/3396 for doing that setup.